.circleci/config.yml

```diff
@@ -296,6 +296,8 @@ jobs:
       - run:
           name: 🔎 run unit tests
           command: |
+              # unfortunately testthat does not and will not support returning a status
```

Comment:
```suggestion
              # unfortunately test that does not and will not support returning a status
```

Reply:
@Marc-Andre-Rivet That's actually the name of the R package, so it's not a typo:

https://cran.r-project.org/web/packages/testthat/index.html